application/client/src/app/activity/activity.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, OnInit } from '@angular/core';
+
+import { Activity } from '../activity';
+import { ActivityService } from '../activity.service';
+
+@Component({
+  selector: 'app-activity',
+  templateUrl: './activity.component.html',
+  styleUrls: ['./activity.component.css']
+})
+export class ActivityComponent implements OnInit {
+  activities: Activity[] = [];
+  activity!: Activity;
+
+  constructor(private activityService: ActivityService) { }
+
+  ngOnInit(): void {
+    this.getActivities();
+  }
+
+  getActivities(): Array<Activity> {
+    this.activityService.getActivities()
+    .subscribe(activities => this.activities = activities);
+
+    console.log(this.activities)
```

Comment:
This won't actually log anything... I will be pushing a change tonight that will be a better way to do this.